.github/workflows/release.yml

```
@@ -48,13 +45,62 @@ jobs:
+    runs-on: ubuntu-latest
+    environment: release
+
+    steps:
+      # Checkout the code
+      - uses: actions/checkout@3d3c42e5aac5ba805825da76410c181273ba90b1 # v7.0.1
+        with:
+          fetch-depth: 0
+
+      # Get the version from the branch name
+      - id: get_version
+        uses: ./.github/actions/get-version
```

actionlint (1.7.12)

[error] description is required in metadata of "Return the version extracted from the branch name" action at "/home/jailuser/git/.github/actions/get-version/action.yml"

(action)